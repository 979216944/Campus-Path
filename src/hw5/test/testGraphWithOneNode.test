# Tests that a node can be added.

# Create a graph
CreateGraph g

# Add a node
AddNode g n

# Print the nodes in the graph
ListNodes g
ListChildren g n