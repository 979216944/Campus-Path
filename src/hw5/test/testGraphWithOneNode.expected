# Tests that a node can be added.

# Create a graph
created graph g

# Add a node
added node n to g

# Print the nodes in the graph
g contains: n
the children of n in g are: